apps/client/src/components/compounds/SetTarget/index.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react'
+import { Text, Input, Button, Card } from 'components'
+import styles from './setTarget.module.scss'
+import CloseIcon from './ClosecloseIcon.svg?react'
+import ArrowDown from './arrow_drop_downdownArrow.svg?react'
+
+interface SetTargetProps {
+  onClose?: () => void
+}
+const SetTarget: React.FC<SetTargetProps> = ({ onClose }) => {
+  const [visible, setVisible] = useState(false)
+  const [selectedFrequency, setSelectedFrequency] = useState('')
+
+  const toggleVisibility = () => {
+    setVisible((prevVisible) => !prevVisible)
+  }
+
+  const handleFrequencySelection = (frequency: string) => {
+    setSelectedFrequency(frequency)
+    setVisible(false)
+  }
+
+  return (
+    <div>
+      <Card className={styles.setTargetContainer}>
+        <div className={styles.setTargetContainerCloseIcon} onClick={onClose}>
+          <CloseIcon />
+        </div>
+        <div className={styles.setTargetContainerText}>
+          <Text content={'Set a Target'} size={'Subtext'} />
+          <Text
+            className={styles.setTargetContainerTextCreate}
+            content={
+              'Create and track multiple saving goals to achieve your financial targets'
+            }
+          />
+        </div>
+        <div className={styles.setTargetContainerInput}>
+          <form>
+            <Input label={'Target'} placeholder={'Type here'} type={'text'} />
+            <Input label={'Target Amount'} placeholder={'Numbers only'} />
+            <div className={styles.setTargetContainerInputFrequency}>
+              <ArrowDown
+                className={styles.setTargetContainerInputArrow}
+                onClick={toggleVisibility}
+              />
+              <Input
+                label={'Frequency'}
+                placeholder={'Pick your frequency'}
+                value={selectedFrequency}
+                onClick={toggleVisibility}
+              />
+              {visible && (
+                <div className={styles.setTargetContainerInputHidden}>
+                  <Text
+                    content="Daily"
+                    onClick={() => handleFrequencySelection('Daily')}
+                  />
+                  <Text
+                    content="Weekly"
+                    onClick={() => handleFrequencySelection('Weekly')}
+                  />
+                  <Text
+                    content="Monthly"
+                    onClick={() => handleFrequencySelection('Monthly')}
+                  />
+                </div>
+              )}
```

Comment:
The ability to translate UI is very important.
This is a select/dropdown UI component, why write the code like this?

We have a select component and you installed react-select, is there a reason you aren't using any of them here?

Reply:
The select component wasn't built in a way that could allow/permit a custom feature and react-select select was allowing my custom styling especially the width to over-ride the default width of react-select that was why i didn't use either of them but didn't remember to remove the library from the project.

Reply:
regardless, your code representation of the UI is wrong.

react select is very customizable, I believe it's possible to style the width 
https://react-select.com/styles

Reply:
besides, we already have a Select component in the codebase, if you can't modify any of the external libraries, then use our default